### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by f00se**
#Basic setup

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,11 +1,12 @@
+# frozen_string_literal: true
```
Comment (PoslinskiNet) [inline Gemfile]: Turn off this setting in rubocop and remove those comments.

Comment (f00se) [inline Gemfile]: Removed

**Comment by f00se** [comment]
Ready for next feedback.

**Comment by PoslinskiNet** [comment]
Move configuration to .example.yml files and remove application.yml and database.yml from the repository (and add to .gitignore)

File: config/application.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Add configuration values here, as shown below.
```
Comment (PoslinskiNet) [inline config/application.yml]: This file should be here :)

File: config/database.yml [hunk 3]
```diff
@@ -5,21 +5,24 @@
 #   gem 'sqlite3'
```
Comment (PoslinskiNet) [inline config/database.yml]: This file should be here :)

File: config/initializers/check_env_vars.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+def compare_configs(config1, config2)
```
Comment (PoslinskiNet) [inline config/initializers/check_env_vars.rb]: Add info to readme about project setup - so someone will know how to run project from scratch. Always if there will be some change - keep it updated.

File: bin/update [hunk 5]
```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env ruby
+
```
Comment (PoslinskiNet) [inline bin/update]: Remove those extra lines in the files if they not introduce any new value.

**Pull request merged**